lang/LangSource/SC_CLIOptions.cpp

```diff
@@ -0,0 +1,277 @@
+            po::bool_switch(&terminal_options.mDaemon)->default_value(terminal_options.mDaemon),
+            "Enter daemon mode"
+        )
+        (
+            "heap-growth,g",
+            po::value<string>()->default_value(convert_memory_integer(terminal_options.mMemGrow))->notifier([&](const string &heapGrowth) {
+                terminal_options.mMemGrow = parse_memory_string(heapGrowth);
+            }),
+            "Set heap growth size (allows k, m and g as suffix multipliers)"
+        )
+        (
+            "yaml-config,l",
+            po::value<string>(&terminal_options.mLibraryConfigFile)->notifier([&](const string &yaml_config) {
+                terminal_options.mLibraryConfigFile = yaml_config;
+                SC_LanguageConfig::setConfigPath(yaml_config);
+            }),
+            "Set yaml config"
+        )
+        (
+            "heap-size,m",
+            po::value<string>()->default_value(convert_memory_integer(terminal_options.mMemSpace))->notifier([&](const string &heapSize) {
+                terminal_options.mMemSpace = parse_memory_string(heapSize);
+            }),
+            "Set initial heap size (allows k, m and g as suffix multipliers)"
+        )
+        (
+            "call-run,r",
+            po::bool_switch(&terminal_options.mCallRun)->default_value(terminal_options.mCallRun),
+            "Call Main.run on startup"
+        )
+        (
+            "call-stop,s",
+            po::bool_switch(&terminal_options.mCallStop)->default_value(terminal_options.mCallStop),
+            "Call Main.stop on shutdown"
+        )
+        (
+            "port,u",
+            po::value<int>(&terminal_options.mPort)->default_value(terminal_options.mPort)->notifier([&](int port) {
+                if (port < 0 || port > 65535) {
+                    cerr << "UDP port must be in range between 0-65535, but was " << port << endl;
+                    mParsingFailed = true;
+                }
+            }),
+            "Set UDP listening port"
+        )
+        (
+            "ide,i",
+            po::value<string>()->notifier([&](string ide_name) {
+                SC_Filesystem::instance().setIdeName(ide_name);
+            }),
+            "Set IDE name"
+        )
+        (
+            "standalone,a",
+            po::bool_switch(&terminal_options.mStandalone)->default_value(terminal_options.mStandalone),
+            "Standalone mode (exclude SCClassLibrary and user and system Extensions folders from search path)"
+        )
+    ;
+    // clang-format on
+
+    return description;
+}
+
+po::options_description CLIOptions::build_language_config_description() {
+    po::options_description description("Language config options (overwrites provided yaml entries)");
+
+    // clang-format off
+    description.add_options()
+        (
+            "include-path",
+            po::value<vector<string>>(),
+            "class library path to be included for searching (allows multiple mentions)"
```

Comment:
"class library path" should start with a capital letter for consistency :monocle_face: